ISISComputingGroup/EPICS-ioc
# Ticket1414 rename lakeshore218

### Description of work

Renamed IOC `LAKESHORE218` to `LKSH218`. No functional changes

### To test

ISISComputingGroup/IBEX#1414

### Acceptance criteria
To test:

* Before switching to this branch, from the GUI start the Lakeshore218 IOC (put it in `SIM` mode, to stop it complaining about not having an instrument) and create a configuration using blocks from the Lakeshore218 IOC. Do the same for a synoptic.
* Switch to this branch, make the LKSH218 IOC, run `make iocstartups` from the top EPICS directory.
* Update your configs and synoptics using the Lakeshore218 to use the new IOC name.
* Restart the instrument
* Open the GUI and see that you can load the configs and synoptics

I have checked `config/` directories for NDXLARMOR and NDXIMAT, and couldn't find any fine where the IOC name needs replacing

I have updated the Release Notes with this ticket (also in the sections specific to each instrument)

---

#### Code Review


### Functional Tests


### Final steps
